action-server/tests/form_helper.py

```diff
@@ -6,6 +6,16 @@
 from rasa_sdk.executor import CollectingDispatcher
 from rasa_sdk.forms import FormAction
 
+PHONE_TRY_COUNTER_SLOT = "daily_ci_enroll__phone_number_error_counter"
+CODE_TRY_COUNTER_SLOT = "daily_ci_enroll__validation_code_error_counter"
+WANTS_CANCEL_SLOT = "daily_ci_enroll__wants_cancel"
+
+INITIAL_SLOT_VALUES = {
+    PHONE_TRY_COUNTER_SLOT: 0,
+    CODE_TRY_COUNTER_SLOT: 0,
+    WANTS_CANCEL_SLOT: False,
+}
```

Comment:
I thought this way too, but this was the simplest way to make it work. And since these are the slots that do have an initial value specified in the domain, inserting them here only makes the tracker closer to what it would contain in a conversation. Depending on their values being None or on their values being the initial ones in another form than daily_CI_enroll have equal chances of causing errors, thus I reached the conclusion that putting them there is rather a neutral solution than an incorrect one.